SalesforceCommerceCloud/pwa-kit
# @W-14139572 - Show discount price on pdp

# Description
This PR created a DisplayPrice component to show discount count price and strighthrough baseprice if applicable. This comp is currently used in PDP and AddToCartModal



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- use-derived-product.js
- pages/product-details/index.jsx
- product-view/index.jsx

# How to Test-Drive This PR

- `npm start` on retail app
- Go to this product: http://localhost:3000/global/en-GB/product/25695436M?color=JJP49XX&size=9SM&pid=701644154707M
- Discount price should show up with basePrice is being strikethrough

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
